io/v5/tape_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestTape_Write(t *testing.T) {
+	db, _ := os.OpenFile(dbFileName, os.O_RDWR|os.O_CREATE, 0666)
+	defer db.Close()
+	defer os.Remove(dbFileName)
+	// defer clean()
```

Comment:
cs violation: remove commented code.

Reply:
if you have to add commented out code then there has to be a comment explaining why it's needed.